content/hacking-atom/sections/cross-platform.asciidoc

```diff
@@ -0,0 +1,58 @@
+---
+title: Cross-platform compatibility
+---
+=== Cross-platform compatibility
+
+Atom runs on a number of platforms and while Electron and Node take care of many of the details there
+are still some considerations to ensure your package works on other operating systems.
+
+==== Symlinks
+File symlinks can be used on Windows by non-Administrators by specifying 'junction' as the type (this argument is ignored on macOS & Linux).
+
+Also consider;
+
+* *Symlinks committed to Git* will not checkout correctly on Windows - _dynamically create what you need with `fs.symlink` instead_
+* *Symlinked directories* are only available to Administrators on Windows - _avoid a dependency on them_
+
+==== Filenames
+* *Reserved filenames* on Windows are `com1`-`com9`, `lpt1`-`lpt9`, `con`, `nul` and `prn` (regardless of extension, e.g. prn.txt is disallowed)
+* *Reserved characters* on Windows are ? \ / < > ? % | : "  _so avoid where possible_
+* *Names with spaces* when passed to the command line;
+** Windows requires you _surround the path with double quotes_  e.g. "c:\my test"
+** macOS and Linux require _a backslash before each space_ e.g. "/my \ test"
```

Comment:
1. Shouldn't be a space before the backslash
1. This sample file name (and the others) should be surrounded with backticks to ensure a literal presentation